src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1089,13 +1089,14 @@ class DocBaseViewer extends BaseViewer {
 
             this.controls.render(
                 <DocControls
+                    annotationColor={this.annotationModule ? this.annotationModule.getColor() : undefined}
```

Comment:
Is this ternary needed? What does `this.annotationModule.getColor()` return by default?

Reply:
We don't need it, it should return bdlBoxBlue by default.